src/app/portlets/shared/dot-content-types-edit/components/layout/content-types-layout.component.html

```diff
@@ -35,9 +35,9 @@ <h4>{{ contentType.name }}</h4>
         header="{{ 'contenttypes.tab.fields.header' | dm }}"
     >
         <div class="content-type__fields-layout" id="content-type-form-layout">
-            <div class="content-type__fields-main" id="content-type-form-main">
+            <dot-portlet-box class="content-type__fields-main">
```

Comment:
As per DMs on Slack, I don't think it's necessary to refactor this component as there might be too many edge cases. I decided to target with a CSS class instead.